layers/+tools/xclipboard/README.org

```diff
@@ -0,0 +1,15 @@
+#+TITLE: xclipboard layer
+
+* Table of Contents                                        :TOC_4_gh:noexport:
+ - [[#description][Description]]
+ - [[#key-bindings][Key Bindings]]
+
+* Description
+This layer adds copy/paste support to the X-clipboard from the terminal.
+
+* Key Bindings
+
+| Key Binding | Description                                 |
+|-------------+---------------------------------------------|
+| ~SPC o y~   | Copy selection to clipboard                 |
+| ~SPC o p~   | Paste clipboard contents at cursor position |
```

Comment:
That is why I didn't explicitly approve this...I don't need to test this..as it works just fine for me....